scripts/debian/postinst

```diff
@@ -58,6 +58,15 @@ if [ "$1" = "configure" ]; then
 fi
 
 if [ "$1" = "configure" ] ; then
+	# Delete all lines referencing /var/lib/vast if it doesn't exist. This works
+	# around restrictive behavior in systemd, which refuses to start the unit if
+	# a ReadWritePath does not exist.
+  if ! [ -d /var/lib/vast ]; then
+    sed -i '/\/var\/lib\/vast/d' /opt/tenzir/lib/systemd/system/tenzir.service
```

Comment:
So this is not yet `tenzir.db` I assume?

Reply:
It is. But the tenzir package should be able to read `/var/lib/vast` if vast was installed before.